schedule.js

```diff
@@ -0,0 +1,21 @@
+import schedule from 'node-schedule';
+import childProcess from 'child_process';
+
+const rule = new schedule.RecurrenceRule();
+rule.minute = 30; // every hour at 30 minutes after the hour
+
+console.log('The scheduler is running…');
+
+schedule.scheduleJob(rule, function(){
+  console.log('Start scheduled job:');
+
+  const forkedProcess = childProcess.fork('./index.js');
```

Comment:
Why don't we just call the main function instead of spawning a new V8 instance with all the memory & CPU costs that entails? 🌱🙂